### PR TITLE
chore: save additional information in the /incoming flag

### PR DESCRIPTION
**Pull request by merlin-northern**
Ticket: None

### CODE REVIEW

**Comment by merlin-northern** [comment]
I am a bit unsure if that will work with the output redirection -- it never worked for me, soe @lluiscampos just feel free to destroy this idea.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -315,10 +315,15 @@ test:acceptance:script:
     - apt update && apt install -yyq awscli
     # Lock the bucket to block concurrent jobs
     - while aws s3 ls s3://$S3_BUCKET_NAME/$S3_BUCKET_REPO_PATH/lock; do
-    -   echo "$S3_BUCKET_REPO_PATH locked, waiting..."
+    -   rm -f /tmp/lock
```
Comment (lluiscampos) [inline .gitlab-ci.yml:318]: This line is unnecessary

**Review by lluiscampos — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (merlin-northern) [reply .gitlab-ci.yml:318]: woops. nice catch! it was supposed to be `/tmp/lock.in` . fixed.

Comment (merlin-northern) [reply .gitlab-ci.yml]: I added one more removal and an echo with waiting, which was there, so I did not try to smuggle too much 😘

**Pull request merged**